console-subscriber/src/aggregator/mod.rs

```diff
@@ -472,6 +472,9 @@ impl Aggregator {
                 task_id,
                 is_ready,
             } => {
+                if self.watchers.is_empty() {
```

Comment:
take it or leave it: maybe worth including a comment here explaining why we drop the poll op if no watches are connected, since this behavior is different from the behavior for other data such as tasks?

Reply:
Will do.